app/models/user.rb

```diff
@@ -154,6 +154,11 @@ class User < ApplicationRecord
     create_communication_preferences
 
     after_confirmation if confirmed?
+    verify_email! if email_status_unverified?
```

Comment:
This is a little trick to ensure that none of the existing unit tests break, otherwise we'd have to stub the verification call in any unit tests that creates a user and runs background jobs

Reply:
We could also just set verified to true in the main user factory?